src/Uno.UWP/System/Launcher.Android.SpecialUris.cs

```diff
@@ -96,6 +96,11 @@ private static bool HandleSettingsUri(Uri uri)
 			}
 
 			var intent = new Intent(launchAction ?? Settings.ActionSettings);
+			if (launchAction == Settings.ActionAppNotificationSettings)
```

Comment:
Please add a manual test for this use case in samples app (I think there might already be a `LauncherTests` sample or something, so just add a specific button + description and mark it as manual test